project/plugins.sbt

```diff
@@ -6,3 +6,6 @@ addSbtPlugin("pl.project13.scala" % "sbt-jmh"                % "0.3.4")
 addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin" % "1.0.0")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage"          % "1.5.1")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"           % "2.3")
+
+addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject"      % "0.6.0")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"                   % "0.6.23")
```

Comment:
`0.6.25` is newest I think

Reply:
yup you'll need that to be 2.13.0-M5 ready